tar/chunks/chunks_directory.c

```diff
@@ -362,6 +362,46 @@ chunks_directory_write(const char * cachepath, RWHASHTAB * HT,
 }
 
 /**
+ * chunks_directory_exists(cachepath):
+ * Return 0 if the /directory file exists within ${cachepath}, 1 if it does
```

Comment:
Seems like it would make more sense to return 1 for `_exists` if the file exists?